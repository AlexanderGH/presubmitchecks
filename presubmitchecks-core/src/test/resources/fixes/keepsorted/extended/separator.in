Matches Google behavior by default:
    // keep-sorted-test start
    1,
    3, // three
    2
    // keep-sorted-test end
    // keep-sorted-test start
    1,
    3,
    2 // two
    // keep-sorted-test end
    // keep-sorted-test start
    c,
    b,
    a
    // keep-sorted-test end

Handles blank lines:
    // keep-sorted-test start
    ".java",

    ".yml",
    // keep-sorted-test end