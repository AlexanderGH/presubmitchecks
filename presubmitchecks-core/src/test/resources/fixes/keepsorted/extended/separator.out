Matches Google behavior by default:
    // keep-sorted-test start
    1,
    2
    3, // three
    // keep-sorted-test end
    // keep-sorted-test start
    1,
    2 // two,
    3
    // keep-sorted-test end
    // keep-sorted-test start
    a,
    b,
    c
    // keep-sorted-test end

Handles blank lines:
    // keep-sorted-test start
    ".java",
    ".yml",
    // keep-sorted-test end
